projects/microsoft.com/azure-cli/package.yml

```diff
@@ -0,0 +1,28 @@
+distributable:
+   url: https://github.com/Azure/azure-cli/archive/refs/tags/azure-cli-{{version}}.tar.gz
+   strip-components: 1
+
+versions:
+  github: Azure/azure-cli/tags
+  strip: /^azure-cli-/
+
+dependencies:
+  openssl.org: 1.1
+  python.org: ^3.10
+  sourceware.org/libffi: '*'
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    rust-lang.org: '*'
+    freedesktop.org/pkg-config: '*'
+  script: |
+    for venv in azure-cli{-telemetry,-core,}; do
+      SRCROOT={{prefix}}/../src-v{{version}}/src/"$venv" python-venv.sh "{{prefix}}"/bin/az
```

Comment:
I _think_ `{{prefix}}/../src-v{{version}}/src/"$venv"` is the same thing as `./src/"$venv"`.

Reply:
throws an error if i do that

```
Error: invalid absolute path: ./src/azure-cli-telemetry
    at new Path (file:///opt/tea.xyz/src-v0.24.9/src/vendor/Path.ts:37:19)
    at from_env (file:///opt/tea.xyz/src-v0.24.9/src/app.ts:175:22)
    at injection (file:///opt/tea.xyz/src-v0.24.9/src/app.ts:165:16)
    at file:///opt/tea.xyz/src-v0.24.9/src/app.ts:34:36
```

Reply:
Hm. Guess that only works for TEA_PANTRY_PATH. Other packages (like `tea.xyz/gx/cc`) suggest that $SRCROOT might be set by default. I don't know if this better:
```sh
_SRCROOT="$SRCROOT"
for ......
   SRCROOT="$_SRCROOT" ...
...
```

That's not much of an improvement, but it protects against future changes.

Reply:
I don't really understand this `_SRCROOT="$SRCROOT"` but it works :)

Reply:
Ha. Captures the original `$SRCROOT` before we start overwriting it.